SparkPost/2web2ui
# FE-1137 - Add Hibana feedback button, add supporting tests

[FE-1137](https://jira.int.messagesystems.com/browse/FE-1137)

### What Changed
- Adds new 'Give Feedback' button for Hibana only that opens a Pendo guide
- The button does not render if HIbana is disabled or `window.pendo` does not exist

### How To Test
1. Sign in to the app, swap to Hibana
2. Verify the "Give Feedback" button renders in the bottom right of the viewport
3. Click on the button and verify the Pendo guide opens

### To Do
- [x] **NOTE** Hold off on deployment until content is approved - will hear back shortly: https://msys.slack.com/archives/CT73SK5NX/p1594840197111400?thread_ts=1594824293.103500&cid=CT73SK5NX
